extensions/spotify-player/src/client/spoti.ts

```diff
@@ -1,1669 +0,0 @@
-// Type definitions for The Spotify Web API (including changes March 29th 2016)
```

Comment:
I imported this file instead of inlining it. Does this make sense or was there a reason for doing this previously?